ccvm_simulators/post_processor/README.md

```diff
@@ -0,0 +1,19 @@
+# Post Processor
```

Comment:
```suggestion
# Post-processor
```

We should be hyphenating this everywhere in our documentation (we use underscores in the code since hyphens are either not allowed of not conventional depending on context).
Thanks @MarkoB4t1qbit for catching this during our discussion!

Reply:
Thanks. Done.